airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -185,7 +184,8 @@ def _parse_and_handle_errors(response) -> Union[MutableMapping[str, Any], List[M
             message = response.json().get("message")
 
         if response.status_code == HTTPStatus.FORBIDDEN:
-            raise HubspotAccessDenied(message, response=response)
+            """ Once hit the forbidden endpoint, we return the error message from response. """
```

Comment:
Maybe move a validation of the 429 code to a backoff approach?

Reply:
There is no issues with `validation of the 429 code` here, it's handled as expected. No need to change this logic.